shutdown_costmaps: false

controller_frequency: 5.0
controller_patience: 3.0


planner_frequency: 1.0
planner_patience: 30.0

oscillation_timeout: 30.0
oscillation_distance: 0.2

max_planning_retries: 1

base_local_planner: "dwa_local_planner/DWAPlannerROS"

base_global_planner: "turtlebot_planner/Planner" 
